### ISSUE TITLE
Video Reader sequence start and end

### ISSUE THREAD

**Issue by Pedrexus** [NVIDIA/DALI#5200]
### Describe the question.

Hello all,

The video reader has a `sequence_length` and `stride` parameters, but is there any way to identify which part of the video I am getting in a batch? It happens that I have long videos with one label for each frame, and using the `get_property` functions gives me repeated filepaths per batch, but no way to identify each frame.

Is there a way to retrieve this kind of info with DALI's video reader?

I found [this issue](https://github.com/NVIDIA/DALI/issues/4808) using external_source, but it doesn't seem to have a solution.

### Check for duplicates

- [X] I have searched the [open bugs/issues](https://github.com/NVIDIA/DALI/issues) and have found no duplicates for this bug report

**Comment by JanuszL** [comment]
Hi @Pedrexus,

Thank you for reaching out. Please check the `enable_frame_num` parameter so you get the frame number of the first frame in the returned sequence. Then you can write custom code to map the number to the label.

**Comment by Pedrexus** [comment]
Hello @JanuszL,

thank you for the help. I was trying to use the experimental video reader, which does not seem to support it.

**Comment by JanuszL** [comment]
Yes, indeed this is one of the functionalities that miss the parity compared to the old reader.
